app/Insightly/Listeners/UpdateContact.php

```diff
@@ -47,6 +54,48 @@ public function handle(ContactUpdated $contactUpdated): void
         );
     }
 
+    private function handleContactEmailChange(Contact $contact): void
```

Comment:
I think when the email is changed other properties like first name and last name could also change. But this is not handled?

Reply:
Should this also happen on the create then? If someone creates a contact with an email address that already exists in Insightly, should the name also be updated?